backend/models/user.go

```diff
@@ -18,6 +18,8 @@ type User struct {
 	Mnemonic          string    `json:"-" gorm:"column:mnemonic"`
 	SSHKey            string    `json:"ssh_key"`
 	Debt              float64   `json:"debt"`
+	Sponsored         bool      `json:"sponsored"`
+	AccountAddress    string    `json:"account_address" gorm:"column:address"`
```

Comment:
why the column is called address instead of account_address, why enforcing ambiguity?